index.js

```diff
@@ -11,8 +11,10 @@ var getDocument = require('get-document');
 
 module.exports = getWindow;
 
+var inNode = require('detect-node');
+
 // old-IE fallback logic: http://stackoverflow.com/a/10260692
-var needsIEFallback = !!document.attachEvent && window !== document.parentWindow;
+var needsIEFallback = !inNode && !!document.attachEvent && window !== document.parentWindow;
```

Comment:
This is what it takes to be able to run in node—not assuming existence of `window` and `document` globals.